Lab2-SyntaxAnalyzer/src/Grammar.hpp

```diff
@@ -209,6 +218,8 @@ struct LR1Item
     Word after_dot;        
     set<Symbol> lookahead;
 
+    LR1Item () : LR1Item("", {}) {}
```

Comment:
nisam sig jel potrebno ovdje eksplicitno definirat konsrtuktor mislim da bi LR1Item(){} po defaultu napravio istu stvar al u svakom sluc ne skodi

Reply:
za to mi je isto javljalo pogresku nesto nesto empty constructor, pa sam mislio da je to mozda ono kj si pricao da je dobro imat prazan konstruktor